libpod/options.go

```diff
@@ -259,7 +259,7 @@ func WithRootFSFromPath(path string) CtrCreateOption {
 // If useImageConfig is specified, image volumes, environment variables, and
 // other configuration from the image will be added to the config
 // TODO: Replace image name and ID with a libpod.Image struct when that is finished
-func WithRootFSFromImage(imageID string, imageName string, useImageConfig bool) CtrCreateOption {
+func WithRootFSFromImage(imageID string, imageName string, mountLabel string, useImageConfig bool) CtrCreateOption {
```

Comment:
Can you add mount label as a separate function? `WithSELinuxMountLabel(mountLabel string) CtrCreateOption`

Reply:
Fixed